### DISCUSSION TITLE
Proposal: Fork of Packer

### DISCUSSION THREAD

**Discussion by MrGrymReaper** [openbao/openbao#78] [General]
As you may or may not know that Hashicorp has been relicensing at least some of its open source software, so that it is now under the Business Source License version 1.1. There's the OpenTofu fork of Terraform and OpenBao fork of Vault, however I'm wondering whether there's any fork of Packer? If there isn't any are there anyone involved in OpenBao or present here who would be willing to also fork the Packer software so that it can continue to be OSS licensed?

**Comment by naphelps** [comment]
@MrGrymReaper To my knowledge there is not a fork of Packer, nor any direct alternatives. I would say any Packer forks would face the same issues as OpenBao, but more intense. You going to need a home for the project (preferably a foundation of some kind), and you are going to need backing.

**Comment by joewxboy** [comment]
@MrGrymReaper 
I do not believe there's any use of Packer here, but I've heard someone within the last week asking if there was a fork.  Let me know if it makes sense to put them in contact with you to see if you can drum up enough interest to start a community.
